posthog/exceptions_capture.py

```diff
@@ -1,13 +1,10 @@
-def capture_exception(error=None, sentry_scope=None, **sentry_scope_kwargs):
-    from sentry_sdk import capture_exception as sentry_capture_exception
+def capture_exception(error=None, properties=None):
     from posthoganalytics import api_key, capture_exception as posthog_capture_exception
     import structlog
 
     logger = structlog.get_logger(__name__)
 
-    sentry_capture_exception(error, scope=sentry_scope, **sentry_scope_kwargs)
```

Comment:
@dmarticus I dug a little further and it looks like removing this line is what is causing the number of query assertions to fail under the hood. Don't exactly know why but it feels like a pretty safe change to make. Going to update the test and merge this